tests/system/action/poll/test_update.py

```diff
@@ -386,3 +386,52 @@ def test_update_permissions(self) -> None:
             {"title": "test_title_Aishohh1ohd0aiSut7gi", "id": 1},
             Permissions.Assignment.CAN_MANAGE,
         )
+
+    def test_update_entitled_users_at_stop_error(self) -> None:
+        response = self.request(
+            "poll.update",
+            {
+                "entitled_users_at_stop": [
+                    {
+                        "voted": True,
+                        "present": True,
+                        "user_id": 2,
+                        "vote_delegated_to_user_id": None,
+                    },
+                    {
+                        "voted": True,
+                        "present": False,
+                        "user_id": 3,
+                        "vote_delegated_to_user_id": 2,
+                    },
+                ],
+                "id": 1,
+            },
+            internal=False,
+        )
+        self.assert_status_code(response, 400)
+        assert (
+            "data must not contain {'entitled_users_at_stop'} properties"
+            in response.json["message"]
+        )
+
+    def test_update_entitled_users_at_stop(self) -> None:
+        user_list = [
```

Comment:
There is no logic check, although it may be a good idea to look into if the things that were changed were actually the ones that are supposed to be changed by the merge. Maybe that should be done for the other changed actions as well...

Reply:
okay